backend/src/main/java/ai/verta/modeldb/configuration/EnabledMigration.java

```diff
@@ -27,7 +27,7 @@ public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata)
     return isMigration();
   }
 
-  public static class InversedEnabledMigration extends EnabledMigration {
+  public static class DisabledMigration extends EnabledMigration {
```

Comment:
*please* don't introduce a class hierarchy for this. The logic is so simple, just repeat it if you need to.

Reply:
Also, would personally prefer this were named "MigrationDisabled"